scripts/pick_one.coffee

```diff
@@ -0,0 +1,14 @@
+# Description:
+#   Example scripts for you to examine and try out.
+#
+# Command:
+#   Hobbes: pick one: Sitar, Zoi's, Tikkaway
+#   Dan: Zoi's
+
+module.exports = (robot) ->
+
+  robot.respond /pick one: (.*)$/i, (msg) ->
+    choiceText = msg.match[1]
+    choices = choiceText.split(",")
+    choice = msg.random choices
+    msg.reply choice
```

Comment:
You mean inline the 'choice' var, yes? I agree. I left it because I was
going to add stuff around it, but wanted to ship something small first. And
because it's hella tricky to add a local var.
On Mar 20, 2015 12:09 AM, "Tom Reznick" <notifications@github.com> wrote:

> In scripts/pick_one.coffee
> <https://github.com/treznick/ContinuityHubot/pull/2#discussion_r26817710>:
>
> > @@ -0,0 +1,14 @@
> > +# Description:
> > +#   Example scripts for you to examine and try out.
> > +#
> > +# Command:
> > +#   Hobbes: pick one: Sitar, Zoi's, Tikkaway
> > +#   Dan: Zoi's
> > +
> > +module.exports = (robot) ->
> > +
> > +  robot.respond /pick one: (.*)$/i, (msg) ->
> > +    choiceText = msg.match[1]
> > +    choices = choiceText.split(",")
> > +    choice = msg.random choices
> > +    msg.reply choice
>
> we can probably just do msg.reply msg.random choices here, but I may be
> wrong about that. My [image: :coffee:] script is [image: :poop:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/treznick/ContinuityHubot/pull/2/files#r26817710>.
>